src/__tests__/commentsTests/commentComponents.js

```diff
@@ -38,6 +38,10 @@ describe("All components", () => {
       }
     });
   });
+  it("component receives props ", () => {
+    component.setProps({ comments: { results: { map }, thread: "" } });
+    expect(component.exists()).toEqual(true);
+  });
```

Comment:
Could you also expect something to happen, on the previous tests you wrote in the `describe("All components"`

Reply:
@LehruAngela,I have made the changes